lib/salus/scanners/report_node_modules.rb

```diff
@@ -136,12 +136,10 @@ def record_dependencies_from_package_json(record_modules: true)
         # Resolve the source if possible from .npmrc
         source = if name.start_with?('@')
                    sections = name.split(%r{\/})
-                   name = sections[1]
```

Comment:
Previously we would be removing the prefix of the name in how we were parsing names that started with @. E.g `@nerv-hq/control-system` would become `control-system`. If there wasn't any reasoning for the previous logic the additional information in the name could be useful.